iis-wf/iis-wf-referenceextraction/src/test/java/eu/dnetlib/iis/wf/referenceextraction/softwareurl/CachedWebCrawlerJobTest.java

```diff
@@ -86,7 +86,7 @@ public void before() throws Exception {
     
     @After
     public void after() throws IOException {
-        FileUtils.deleteDirectory(workingDir);
+//        FileUtils.deleteDirectory(workingDir);
```

Comment:
I'll drop this line if we don't need it.

Reply:
Oh, we need this line, I've commented it for development and committed by mistake.